docs/NativeModulesAndroid.md

```diff
@@ -18,7 +18,7 @@ This guide will use the [Toast](http://developer.android.com/reference/android/w
 We start by creating a native module. A native module is a Java class that usually extends the `ReactContextBaseJavaModule` class and implements the functionality required by the JavaScript. Our goal here is to be able to write `ToastAndroid.show('Awesome', ToastAndroid.SHORT);` from JavaScript to display a short toast on the screen.
 
 ```java
-package com.facebook.react.modules.toast;
+package com.example.modules.toast;
```

Comment:
@gs-akhan No, it doesn't. I can easily have it a completely different package and import the class from that package.

Since you've created the file in the same folder, you need to have same package name. But say you have a subfolder named modules, and place it there, then the package name will be different. That doesn't mean you won't be able to use it.

If you want an example, look at any react native module, say https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java . The package name in that file is `com.facebook.react.modules.websocket`, but it's stil being used by https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/shell/MainReactPackage.java which has the package name `com.facebook.react.shell`, which is ultimately used by your app, which has the package name, say `com.awesomeinc.myapp`. So package name doesn't matter.